### ISSUE TITLE
Writing tests in Unity

### ISSUE THREAD

**Issue by ghost** [DemoProductions/shmup#47]
https://unity3d.com/learn/tutorials/topics/production/unity-test-tools

**Comment by ghost** [comment]
Going to try adding a few tests. Haven't done any test automation with Unity yet. Hopefully will not add too many dependencies

**Comment by flip40** [comment]
Yeah, I need to watch that video. I'm curious as to how testing in Unity actually works..
